launcher/ui/pages/instance/ManagedPackPage.cpp

```diff
@@ -62,8 +62,11 @@ ManagedPackPage::ManagedPackPage(BaseInstance* inst, InstanceWindow* instance_wi
 
     // NOTE: GTK2 themes crash with the proxy style.
     // This seems like an upstream bug, so there's not much else that can be done.
-    if (!QStyleFactory::keys().contains("gtk2"))
-        ui->versionsComboBox->setStyle(new NoBigComboBoxStyle(ui->versionsComboBox->style()));
+    if (!QStyleFactory::keys().contains("gtk2")){
+        auto comboStyle = new NoBigComboBoxStyle(ui->versionsComboBox->style());
+        comboStyle->setParent(APPLICATION); // make sure this gets cleaned up (setting to simply `this` causes it to be freed too soon)
```

Comment:
does this make sense as a fix though? for sure, it will clean up the leaked memory when we are exiting the application, but while it is still running, if you open the page multiple times, the leak will still be there.

Reply:
hmm. perhaps it needs to be made into a singleton. there is something about QStyle objects that prevent it from being freed even after the widget that used them is freed...